package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua

```diff
@@ -1,3 +1,27 @@
+local cbi = require 'luci.cbi'
 local i18n = require 'luci.i18n'
+local site = require 'gluon.site_config'
+local util = require 'luci.util'
+local gluon_luci = require 'gluon.luci'
+local sysconfig = require 'gluon.sysconfig'
+local pretty_hostname = require 'pretty_hostname'
 
-return function () luci.template.render_string(i18n.translate('gluon-config-mode:reboot')) end
+local uci = luci.model.uci.cursor()
+
+local pubkey = util.trim(util.exec("/etc/init.d/fastd show_key " .. "mesh_vpn"))
```

Comment:
You can't do this, gluon-config-mode-core must not expect that fastd is installed on the router. We already have gluon-config-mode:pubkey for dealing with VPN-specific output.